ObjCVoiceCallKitQuickstart/ViewController.m

```diff
@@ -200,29 +200,32 @@ - (void)callDidConnect:(TVOCall *)call {
     [self routeAudioToSpeaker];
 }
 
-- (void)callDidDisconnect:(TVOCall *)call {
-    NSLog(@"callDidDisconnect:");
-
-    [self performEndCallActionWithUUID:call.uuid];
-
-    self.call = nil;
-    self.callKitCompletionCallback = nil;
+- (void)call:(TVOCall *)call didFailToConnectWithError:(NSError *)error {
+    NSLog(@"Call failed to connect: %@", error);
     
-    [self.placeCallButton setTitle:@"Place Outgoing Call" forState:UIControlStateNormal];
-    
-    [self toggleUIState:YES];
+    [self performEndCallActionWithUUID:call.uuid];
+    [self callDisconnected];
```

Comment:
We should fire `self.callKitCompletionCallback(NO)` within this callback, just like we did prior to this PR.